.github/policies/moderatorTriggers.yml

```diff
@@ -98,8 +98,6 @@ configuration:
               repo names). Alternatively, you can open a Feedback Hub issue and attach them there. 
               If you use Feedback Hub, please paste the URL at the bottom of the report here so we 
               can easily find it. 
-      - removeLabel:
-          label: Needs-Triage
```

Comment:
Should we add `Needs-Author-Feedback` since the log message is asking the author to provide input?

Reply:
I think no. Needs-Author-Feedback puts it on the "if we don't hear back, we need to close because not enough info" path, but there have been various times when we know the issue is there and (more) logs would just be helpful.